index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav class="menu">
+        <h1 class="logo">MIRNA</h1>
+        <img src="./image/Enabled.png" alt="hamburger icon" class="hamburger_icon">
+    </nav>
+    <header class="header">
+        <h2 class="header_title">I'm Mirna, welcome to my portfolio!</h2>
+        <p>I'm a software developer!
+             I can help you build a product ,
+              feature or website Look through 
+              some of my work and experience! 
+              If you like what you see and have 
+              a project you need coded, 
+              don't hestiate to contact me.</p>
+        <h3 class="final_msj">LET'S CONNECT</h3>
```

Comment:
- [ ]  Please use an `a` or `button `tag instead of `h3` tag. and follow the button styling: Active, Hover and Pressed (see design).
Apart from being semantically correct, the `a `tag will let you connect to other sections of the website, using href.